desktop/apps/article2/components/App.js

```diff
@@ -1,12 +1,11 @@
-import PropTypes from 'prop-types'
-import * as React from 'react'
-import components from '@artsy/reaction-force/dist/Components/Publishing'
-import InfiniteScrollArticle from './InfiniteScrollArticle'
-import { data as sd } from 'sharify'
+import ArticleModel from 'desktop/models/article.coffee'
 import EditButton from 'desktop/apps/article2/components/EditButton'
+import InfiniteScrollArticle from './InfiniteScrollArticle'
+import PropTypes from 'prop-types'
+import React from 'react'
 import SuperArticleView from 'desktop/components/article/client/super_article.coffee'
-import ArticleModel from 'desktop/models/article.coffee'
-const { Article } = components
+import { Article } from '@artsy/reaction-force/dist/Components/Publishing'
+import { data as sd } from 'sharify'
```

Comment:
Alpha on imports (highlight "sort natural" or related in editor)  -- Order logic follows: 
```js
*
_ 
A
B
a
b
{ A }
{ B } 
{ a }
{ b }
```